burst-apps-team/burstcoin
# Adding Phoenix again

Adding Phoenix as alternative wallet...BRS should work now

To test with Phoenix wallet you need:

> NodeJS 14+ installed 

go into `./ci` folder and run `./updatePhoenix.sh` - this will pull latest phoenix release from git and add to `html/ui` 

start brs and it should work

![classic-phoenix](https://user-images.githubusercontent.com/3920663/113064135-d16d9200-918c-11eb-98f6-7257815463dc.gif)
